### PR TITLE
Condense patient menu

### PR DESCRIPTION
**Pull request by YandanaB**
Just edits the flow of the patient.py file so that can see less options ("Edit Personal Info", "Record Mood of the Day", "Display Previous Moods", "Add Journal Entry", "Read Journal Entries", "Search Exercises", "Log out") if don't have clinician or these options with option of appointments if have clinician. With appointments see options "Book Appointment", "View Appointments", "Cancel Appointment", "Exit appointments"). Exit leads to patient menu. Patients with disabled accounts can login and logout only to see a message to contact admin. I changed the retry option so that quit is now an option in patient main menu. I tested all combinations but now there are so many options and nested options that it might be a good idea for you test them quickly as well. Thanks.

**NOTE** : Must logout. Can clean. Can switch to while loop.

When there is merge conflict with utilities, accept phil's incoming change. I just copied utilities off his branch.

### CODE REVIEW

**Review by davidve0206 — CHANGES_REQUESTED** [review]

File: modules/patient.py [hunk 1]
```diff
@@ -380,94 +380,137 @@ def flow(self):
...
-            next_step = display_choice(
-                "Would you like to:",
-                [
-                    "Retry the same action",
-                    "Go back to the main menu",
-                    "Quit",
-                ],
-                choice_str="Your selection: ",
-            )
+                    log_out = input(
```
[the remark this answers is not in the document]
Comment (YandanaB) [reply modules/patient.py]: Yes, that makes sense. I am sorry about that. Have removed things on user accounts that are disabled.

[the remark this answers is not in the document]
Comment (davidve0206) [reply modules/patient.py]: no need to apologise!

File: modules/patient.py [hunk 2]
```diff
@@ -447,6 +475,8 @@ def acting_on_choice(choice):
                         self.search_exercises(keyword)
 
                     case 7:
+                        # if statement here since case 7 would be appointments if
+                        # the patient had a clinician. Otherwise it would be log out.
```
Comment (philipcalderwood) [inline modules/patient.py:479]: is some of this logic made easier by moving to a system where 0 is always quit?

File: modules/patient.py [hunk 3]
```diff
@@ -477,46 +508,63 @@ def acting_on_choice(choice):
...
-                    #    return False
 
+            # Matching choices here outside of the acting on choice function
+            # if the choice is to logout or quit.
             match choice:
                 case 7:
+                    # Option 7 is appointments if there is a clinician. This is already
+                    # put in place in acting on choice function.
+
+                    # If there is no clinician, option 7 is logout
```
Comment (philipcalderwood) [inline modules/patient.py:546]: so keeping it at seven would simplify this into a single conditonal statement right?  `if (clinician_id): ...`

**Review by philipcalderwood — COMMENTED** [review]
Reading overall could we simplify approach by using quit as 0 and only having logout functionality from the main screen? Do we need a logout functionality at all? I think from this PR my vote is for 0 as quit throughout and scrap logout entirely for simplicity. Will put this to group.

**Pull request merged**